src/test/ui/async-await/async-fn-size.rs

```diff
@@ -1,9 +1,10 @@
+// run-pass
+
```

Comment:
```suggestion
// aux-build:arc_wake.rs 
```
(to fix the failure)

Reply:
Thanks. It's fixed. Wonder why it wasn't caught in the test of first commit...